app/services/supplementary_data.py

```diff
@@ -78,6 +96,22 @@ def get_supplementary_data(*, dataset_id: str, unit_id: str, survey_id: str) ->
     raise SupplementaryDataRequestFailed
 
 
+def decrypt_supplementary_data(supplementary_data: MutableMapping) -> Mapping:
+    if encrypted_data := supplementary_data.get("data"):
+        try:
+            decrypted_data = JWEHelper.decrypt(
```

Comment:
Why does this use the `JWEHelper.decrypt` over the `sdc.crypto.decrypter.decrypt` which wraps that method? Authenticator uses the decrypter one so I'm just wondering why this one doesn't, was the check on the tokens not needed here?

Reply:
That method expects a public key to exist as well when decoding, all we need is the decryption part as this isn't signed - just a symmetric key scenario here, unless Mebin's above comment resolves to going back to asymmetric.

Reply:
ahhhh ok - makes sense, thanks 👍